### PR TITLE
mice that chew cables will now be fried

### PR DESCRIPTION
**Pull request by press-mark**
## About The Pull Request
mice that chew through cables will now be thoroughly cooked

## Why It's Good For The Game
you're telling me that mice that chewed through the wires were still raw? smh my head

## Changelog

:cl:
add: mice that chew cables will now be fried
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 1]
```diff
@@ -64,6 +64,8 @@
 			if(toast)
 				M.add_atom_colour("#3A3A3A", FIXED_COLOUR_PRIORITY)
 				M.desc = "They're toast."
+				M.foodtypes &= ~ RAW //it's toast!
+				M.foodtypes |= FRIED
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm:68]: foodtypes is used on examine and during Initialize when making it editable, adding this to the list after you made the food won't actually affect the food, just the description of it, you can look to https://github.com/tgstation/tgstation/pull/68319 to see how that worked out.

**Comment by CPTANT** [comment]
Can you still make organic slurry with the fried mice? It would be rather unfortunate if you can't.

**Comment by Kitfox88** [comment]
You're telling me a wire fried this mice?

File: code/modules/mob/living/simple_animal/friendly/mouse.dm (hunk 1 above)
Comment (coldud13) [reply code/modules/mob/living/simple_animal/friendly/mouse.dm:68]: i'm not sure that i follow

**Comment by coldud13** [comment]
still stuck on this one, not too sure how to implement it like https://github.com/tgstation/tgstation/pull/68319 :/

**Comment by JohnFulpWillard** [comment]
You can generalize the silver slime code to let you send any food flags as an arg, then call it that way
So instead of ``mark_silver_slime_reaction()`` it would be ``give_food_foodflag(FOOD_SILVER_SPAWNED)``, which will also pass through the signal.
It will require some work but you'll get what you want done, and make it easier for people in the future to add their own without the same problems you two had to deal with.

**Comment by SuperNovaa41** [comment]
Feel free to reopen this when you figure it out

**Pull request closed without merging**